service/app/utils/module/RequestIdAddFilter.java

```diff
@@ -26,8 +26,7 @@ public RequestIdAddFilter(Materializer materializer) {
   public CompletionStage<Result> apply(
       Function<Http.RequestHeader, CompletionStage<Result>> nextFilter,
       Http.RequestHeader requestHeader) {
-    long startTime = System.currentTimeMillis();
-
+    MDC.clear();
```

Comment:
looks this is not the right place - pls put a comment whats the problem and how'd you think this will resolve.

Reply:
The problem was, sometimes it was giving previous value, I am not sure why. And one more problem I found now was, In this class sometimes it is setting reqID in MDC, sometimes it not setting. So I moved this MDC into OnRequestHandler